ui/components/component-library/picker-network/picker-network.scss

```diff
@@ -0,0 +1,4 @@
+.mm-picker-network {
+  max-width: 100%;
+  height: 32px;
```

Comment:
I know there is only a small amount of styles here but could we store this in a component tiered css variable?

`--picker-network-height: 32px`

```
height: var(--picker-network-height);
```